sci-libs/coinasl/coinasl-2.0.1.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Ampl Solver Library (ASL)"
+HOMEPAGE="https://github.com/coin-or-tools/ThirdParty-ASL"
+SOLVERS_SHA="64919f75f"
+SRC_URI="https://coin-or-tools.github.io/ThirdParty-ASL/solvers-${SOLVERS_SHA}.tgz
+	https://github.com/coin-or-tools/ThirdParty-ASL/archive/refs/tags/releases/${PV}.tar.gz -> ${P}.tar.gz"
+IUSE="big-goff-32 big-goff-64 static-libs"
```

```suggestion

IUSE="big-goff-32 big-goff-64 static-libs"
```

Do we really need `static-libs`? If there is no use case for it, then lets remove the use flag.